tests/testthat/test-authors.R

```diff
@@ -172,7 +172,6 @@ test_that("we cannot add the same ORCID to more than one author", {
 
 })
 
-
```

Comment:
Sorry,  there is still a whitespace change here.

Reply:
Oh, I think I misunderstood the style

Reply:
I just prefer to avoid whitespace changes in general, even if they "fix" the style.